app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Confy</title>
-    <%= csrf_meta_tags %>
+<head>
+  <title>Confy</title>
+  <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
+  <%= env_stylesheet_link_tag(
+      static: 'application',
+      options: { media: 'all', 'data-turbolinks-track' => true }) %>
+  <%= env_javascript_include_tag(
+      static: 'application',
+      hot: %w(http://localhost:3500/vendor-bundle.js http://localhost:3500/app-bundle.js)) %>
```

Comment:
Weeeell.. yes, as far as I know. Unless you have some other suggestion. This allows hot reloading webpack assets in dev environment. They are served by webpack-dev-server at this URL.